src/examples/hierarchy-browser-basic/example-hierarchy-browser-basic.component.ts

```diff
@@ -44,22 +44,22 @@ export class ExampleHierarchyBrowserBasicComponent {
 			.group({
 				toggle: new Command({
 					execute: function execute(node) {
-						if (node.isVisited) {
+						node.isExpanded = !node.isExpanded;
+
+						if (!node.isExpanded) {
 							return;
 						}
 
-						node.isVisited = true;
-						const length = Math.floor(Math.random() * 9 + 1);
 						const level = node.level + 1;
+						const childrenLength = node.children.length;
+						const exampleChildren = [
+							new Node(`folder [${level},${childrenLength}] `, level, 'group'),
+							new Node(`folder [${level},${childrenLength + 1}] `, level, 'group'),
+							new Node(`file [${level},${childrenLength + 2}] `, level, 'value'),
+							new Node(`file [${level},${childrenLength + 3}] `, level, 'value'),
+						];
 						setTimeout(() => {
```

Comment:
can we remove SetTimeout it was for emulating hard disk, don't think that we need it